## H0   All proportions are equal 
## HA Not all proportions are equal 


In [2]:
import pandas as pd
import numpy as np
from scipy.stats import chi2_contingency

In [10]:
data=pd.read_csv(r"D:\DS PYTHON\Assignment\BuyerRatio.csv")

In [11]:
data

Observed Values  East  West  North  South
0           Males    50   142    131     70
1         Females   435  1523   1356    750

In [12]:
obs=np.array([[50,142,131,70],[435,1523,1356,750]])

In [8]:
obs

array([[  50,  142,  131,   70],
       [ 435, 1523, 1356,  750]])

In [13]:
statistics,pvalue,df,array=chi2_contingency(obs)

In [14]:
statistics

1.595945538661058

In [15]:
pvalue

0.6603094907091882

In [16]:
df

3

In [17]:
array

array([[  42.76531299,  146.81287862,  131.11756787,   72.30424052],
       [ 442.23468701, 1518.18712138, 1355.88243213,  747.69575948]])

### alpha < pvalue 
### 0.05 < 0.66

#### We failed to reject the null hypothesis or accept the null hypothesis

## Statement: All proportions are equal